config/default/block_content.type.uiowa_articles.yml

```diff
@@ -0,0 +1,8 @@
+uuid: cbea77a6-0e50-488a-8d6c-c228b11dc1b9
+langcode: en
+status: true
+dependencies: {  }
```

Comment:
I guessing this has a hard dependency on sitenow_articles, as does sitenow_people.

Reply:
Added.